salt/_pillar/metalk8s_endpoints.py

```diff
@@ -0,0 +1,57 @@
+"""Store data about bootstrap services ip/port in pillar"""
+
+import logging
+
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = 'metalk8s_endpoints'
+
+
+def __virtual__():
+    return __virtualname__
```

Comment:
Strictly speaking this should indicate failure if `kubernetes.show_endpoint` can't be found in `__salt__`, but fine.